dotnet-grad/Models/TestContext.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+
+namespace dotnet_grad.Models
+{
+  public class TestContext : DbContext
+  {
+
+    public DbSet<UserModel> Users { get; set; }
+    public DbSet<OrganisationModel> Organisations { get; set; }
+    public DbSet<AddressModel> Addresses { get; set; }
+
+    public string DbPath { get; }
```

Comment:
If not used outside of this class, it's best to have it as a private member, and if only modified/initialised in the constructor then then readonly. Private members are also underscored as an industry standard to reveal the intention.